CodeEdit/TabBar/TabBarItem.swift

```diff
@@ -331,6 +331,8 @@ struct TabBarItem: View {
                     workspace.closeTabs(after: item.tabID)
                 }
             }
+            // Disable this option when current tab is the last one.
+            .disabled(workspace.selectionState.openedTabs.last?.id == item.tabID.id)
```

Comment:
Will this work when we have split editor layouts?

Reply:
I will take a look on how split works.

Reply:
I think we do not have any window-split functionality for now. And when we are implementing it in the future, we are probably going to split them in different `WorkspaceDocument` to correctly order tabs. The reference `workspace.selectionState.openedTabs` is only related to the workspace that the current tab is located at.